graph/GuidelinesGraph.md

```diff
@@ -324,32 +321,34 @@ The top-level error code must match the HTTP response status code description, c
 | :heavy_check_mark: **MUST** return a 429 Too Many Requests error when the client exceeded throttling limits, and a 503 Service Unavailable error when the service overloaded but the client is within throttling limits.|
 | :ballot_box_with_check: **SHOULD** return a 404 Not Found error if a 403 error would result in information disclosure. |
 
+For additional guidance, see [Error condition responses article](Guidelines-deprecated.md#7102-error-condition-responses).
```

Comment:
I assume that anything referencing the deprecated doc needs to be pulled into an article?